apps/condo/domains/ticket/constants/statusTransitions.js

```diff
@@ -28,6 +28,10 @@ const COUNTRY_RELATED_STATUS_TRANSITIONS = {
 
 const TICKET_ORDER_BY_STATUS = {
     [STATUS_IDS.OPEN]: 100,
+    [STATUS_IDS.IN_PROGRESS]: 200,
```

Comment:
we can create our own status ids for organizations. You need to pay attantion

Reply:
In this case, I think I will make `order = 100` only for the "Open" status, for all others set 200 (not only for default ones)